L/louvain/build_tarballs.jl

```diff
@@ -0,0 +1,39 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "louvain"
+version = v"0.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/KrainskiL/louvain.git", "9e3e081f054e531e32af38fe71d2991f954ea347")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/louvain/
+make -j${nproc}
+mkdir -p ${bindir}
+for exe in hierarchy louvain matrix convert; do
+    mv "${exe}" "${bindir}/${exe}${exeext}"
+done
+"""
```

Comment:
We should also install the license files:
```suggestion
install_license *gpl*.txt
"""
```
:wink:

Reply:
Good point. Thank you.